Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexFormat.java

```diff
@@ -111,7 +111,8 @@ private static String formatTime(TimexProperty timex) {
     private static String formatDate(TimexProperty timex) {
```

Comment:
All this method was reduced to one line in [C#](https://github.com/microsoft/Recognizers-Text/pull/2457/files#diff-4f811f041fb2b829ff5ec51e6b8a43862c8cbcb3a7552965a33b97e1aa561dedR124).

Why do we have all these here?